sokoban/src/App.tsx

```diff
@@ -1,17 +1,23 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import { GameArray } from "./Globals";
 import { Form } from "./components/Form";
 import "./game.css";
+import { handleBoard } from "./player";
+import BackgroundBox from "./assets/images/box.jpg";
+import BackgroundFloor from "./assets/images/floor.jpg";
+import BackgroundPlayer from "./assets/images/player.jpg";
+import BackgroundTarget from "./assets/images/Target.jpg";
+import BackgroundWall from "./assets/images/wall.jpg";
 
-const colors: any = {
-  w: "red",
-  b: "brown",
-  p: "green",
-  "": "burlywood",
+const backgoundImage: any = {
+  w: BackgroundWall,
```

Comment:
There is no need to import images? Just give the link directly in the object:

 backgroundImage {
b: "./assets/images/box.jpg",
p: "./assets/images/floor.jpg",
etc etc
}

Reply:
I have changed, but it does not work for me.
I tried this:
const backgoundImage: any = {
  w: "./assets/images/wall.jpg",
  b: "./assets/images/box.jpg",
  p: "./assets/images/player.jpg",
  tp: "./assets/images/player.jpg",
  "": "./assets/images/floor.jpg",
  t: "./assets/images/Target.jpg",
};